search_knn.go

```diff
@@ -196,6 +197,7 @@ func (r *SearchRequest) UnmarshalJSON(input []byte) error {
 		r.KNN[i].VectorBase64 = temp.KNN[i].VectorBase64
 		r.KNN[i].K = temp.KNN[i].K
 		r.KNN[i].Boost = temp.KNN[i].Boost
+		r.KNN[i].Params = temp.KNN[i].Params
```

Comment:
Was this a regression from the previous change?

Reply:
yes, fixed in a recent commit.